resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/features/BasicExecutorFeatureStepSuite.scala

```diff
@@ -176,26 +176,52 @@ class BasicExecutorFeatureStepSuite extends SparkFunSuite with BeforeAndAfter {
     checkOwnerReferences(executor.pod, DRIVER_POD_UID)
   }
 
+  def withPodNamePrefixRestored(f: => Unit): Unit = {
```

Comment:
Just a question. Do we have a naming convention `XXXRestored` in Apache Spark codebase?

Reply:
might be just a language issue. I will rename it to `withPodNamePrefix`